GOVERNANCE.md

```diff
@@ -0,0 +1,201 @@
+# ESLint-Community Governance
+
+## Core Team
+
+The Core Team are the curators and stewards of the ESLint-Community project . Their key
+responsibility is to evaluate new projects that are proposed to join `@eslint-community`
+and to help out in offering a base level maintenance for all primary projects within
+the organization.
+
+The Core Team should have occasional meetings to discuss the project, with meeting minutes
+being added to [eslint-community/governance][].
+
+The Core Team are together with the ESLint TSC the organization owners and the only ones
+with `admin` access to repositories within the organization.
+
+The Core Team are the only members of the `@eslint-community/core-team` team.
+
+The Core Team and ESLint TSC have a shared `#community-team` chat room, in the ESLint
+Discord, to communicate with one another.
+
+## ESLint TSC
+
+The ESLint TSC are the ultimate caretakers of the ESLint-Community, appoints the Core
+Team and holds the ownership of the `eslint-community-bot` npm account that owns all
+the ESLint-Community npm modules.
+
+The Core Team are together with the ESLint TSC the organization owners and only ones
+with `admin` access to repositories within the organization.
+
+The ESLint TSC are the only members of the `@eslint-community/eslint-tsc` team.
+
+The Core Team and ESLint TSC have a shared `#community-team` chat room, in the ESLint
+Discord, to communicate with one another.
+
+## Collaborators
+
+Collaborators maintain the projects of the ESLint-Community organization.
+
+Collaborators are added on a project by project basis by adding them to a team of
+the same name as its main project repository, eg. `@eslint-community/eslint-plugin`.
+
+A project team is given `write` level access to its repositories.
+
+If multiple repositores are needed by the same project they should all be added to
+the project team.
+
+Both Collaborators and non-Collaborators may propose changes to the source code
+of the projects of the organization. The mechanism to propose such a change is a
+GitHub pull request. Collaborators review and merge (_land_) pull requests
+following the [CONTRIBUTING](CONTRIBUTING.md) guidelines.
```

Comment:
Can mention be made here, or in a stub `CONTRIBUTING.md` document, that breaking changes should be reviewed by team members, if that is indeed the case? That seems to be a reasonable policy. I think it would be good to also explicitly mention that other merges can be handled by a collaborator who is different from the person submitting the PR.

Reply:
Makes a lot of sense 👍

I would also like it if we would do some general guidance decisions on things like new engine ranges, so that contributors can reference that